apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -1,20 +1,36 @@
+import { UserValidationMessage } from "shared/src/modules/users/libs/enums/user-validation-message.enum.js";
```

Comment:
Do not directly import anything from shared. Add this enum to the apps/backend/src/modules/auth/libs/enums/enums.ts file. Import that type there and export it so that you can import the enum into this service. Just like you do with AuthorizationError